packages/api-key-service/src/constants.ts

```diff
@@ -1,3 +1,3 @@
 export enum TABLE_NAMES {
-  API_KEY_TABLE = 'Cloud_Api_Key',
+  API_KEY_TABLE = 'Cloud_Api_Key2',
```

Comment:
Why the table name change? Is it just because the old one needs deleting first?

Reply:
yes :p I didn't have permission and couldn't figure out how to give myself permission to delete the table. This was in the colsole this time!

Reply:
Personally, I would make this come from the config?